lib/pengine/native.c

```diff
@@ -184,14 +184,11 @@ native_add_running(pe_resource_t * rsc, pe_node_t * node, pe_working_set_t * dat
 }
 
 static void
-recursive_clear_unique(pe_resource_t *rsc)
+recursive_clear_unique(pe_resource_t *rsc, gpointer user_data)
```

Comment:
A pcmk__list_foreach_1 function would get rid of the need to do this.  I think there's one other spot in this patch where I made this same kind of change.

Reply:
I don't think it's worth a new name